test/libsolidity/syntaxTests/memberLookup/failed_function_lookup_in_library.sol

```diff
@@ -0,0 +1,9 @@
+library L {
+  function f(uint, uint) {}
+  function f(uint) {}
+}
+contract C {
+  function g() { L.f(1, 2, 3); }
+}
+// ----
+// TypeError: (94-97): Member "f" not found or not visible after argument-dependent lookup in type(library L)
```

Comment:
Added this test case from the original issue.

Reply:
A better question though: cannot we support overloads from libraries?

Reply:
Don't we?

Reply:
The above example is broken and doesn't fullfill the actual test purposes it should.

There is no overload with 3 arguments. The call should be changed to two.

Reply:
Never mind, it was too late.